scripts/cmux-events-stress.py

```
@@ -0,0 +1,653 @@
+            publisher.join(timeout=self.args.publish_timeout)
+            if publisher.is_alive():
+                raise StressFailure(f"publisher did not finish within {self.args.publish_timeout}s")
+            if self.publisher_error:
+                raise StressFailure(self.publisher_error)
+            self.summary["publish_duration_ms"] = rounded_ms(now_ms() - publish_start)
+
+            for thread in threads:
+                thread.join(timeout=self.args.consumer_timeout)
+            alive = [thread.name for thread in threads if thread.is_alive()]
+            if alive:
+                raise StressFailure(f"event consumers did not exit cleanly: {alive}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)